raiden/settings.py

```diff
@@ -44,7 +44,7 @@
 
 DEFAULT_PATHFINDING_MAX_PATHS = 3
 DEFAULT_PATHFINDING_MAX_FEE = 1000
-DEFAULT_PATHFINDING_IOU_TIMEOUT = 500
+DEFAULT_PATHFINDING_IOU_TIMEOUT = 5000000000
```

Comment:
158 Years?

Reply:
hehe changed it to 50000 (so 200h for the pfs to cash in)